src/Factory/AnnotatedServiceAbstractFactory.php

```diff
@@ -1,41 +1,38 @@
 <?php
+
 /**
  * @see https://github.com/dotkernel/dot-annotated-services/ for the canonical source repository
- * @copyright Copyright (c) 2017 Apidemia (https://www.apidemia.com)
- * @license https://github.com/dotkernel/dot-annotated-services/blob/master/LICENSE.md MIT License
  */
 
-declare(strict_types = 1);
+declare(strict_types=1);
 
 namespace Dot\AnnotatedServices\Factory;
 
 use Dot\AnnotatedServices\Annotation\Service;
-use Interop\Container\ContainerInterface;
+use interop\container\containerinterface;
```

Comment:
First, this should not be lowercase - not sure why you changed it.
Second, it should be:
`use Psr\Container\ContainerInterface;`
Please change and test. If I'm wrong, then let's keep the old version:
`use Interop\Container\ContainerInterface;`